aten/src/ATen/native/TensorCompare.cpp

```diff
@@ -115,13 +115,10 @@ Tensor isfinite(const Tensor& self) {
 
 bool is_nonzero(const Tensor& self) {
   auto n = self.numel();
-  AT_ASSERT(n >= 0);
-  if (n == 0) {
-    AT_ERROR("bool value of Tensor with no values is ambiguous");
-  }
-  if (n > 1) {
-    AT_ERROR("bool value of Tensor with more than one value is ambiguous");
-  }
+  TORCH_INTERNAL_ASSERT(n >= 0);
```

Comment:
Why is this there? This would be always true right?

Reply:
I agree. This was translating `AT_ASSERT`, but I think this isn't really needed.